licenses/templates/macros_templates/pdtool_deed.html

```diff
@@ -38,7 +38,7 @@
                 <div id="donation-bar-wrapper" class="donation-bar-wrapper">
                   <div id="donation-bar-inner" class="donation-bar-inner">
                     <p class="donate-text">{% blocktrans %}Help us build a vibrant, collaborative global commons{% endblocktrans %}</p>
-                    <div class="donate-action"><a href="https://us.netdonor.net/page/6650/donate/1?ea.tracking.id=deed-top" class="button donate arrow">{% blocktrans %}Donate{% endblocktrans %}<span class="hide-on-mobile"> {% blocktrans %}Now{% endblocktrans %}</span></a></div>
+                    <div class="donate-action"><a href="https://creativecommons.org/donate?ea.tracking.id=deed-top" class="button donate arrow">{% blocktrans %}Donate{% endblocktrans %}<span class="hide-on-mobile"> {% blocktrans %}Now{% endblocktrans %}</span></a></div>
```

Comment:
Should be: https://www.classy.org/give/313412/#!/donation/checkout?c_src=website&c_src2=Deed-Top